src/FQDNLabel.php

```diff
@@ -258,7 +258,10 @@ public static function getItemsByFQDN($fqdn, $wildcard_search = false)
         foreach (self::getIDsByLabelAndFQDNID($label, $fqdns_id, $wildcard_search) as $class => $IDs) {
             if ($FQDNlabel = getItemForItemtype($class)) {
                 foreach ($IDs as $ID) {
-                    if ($FQDNlabel->getFromDB($ID)) {
+                    if (
+                        $FQDNlabel->getFromDB($ID)
+                        && ($FQDNlabel instanceof CommonDBChild)
```

Comment:
Shouldn't we throw a warning somewhere if we got something that is not a `CommonDBChild`?